packages/react/src/ReactElement.js

```diff
@@ -12,6 +12,8 @@ import ReactCurrentOwner from './ReactCurrentOwner';
 
 const hasOwnProperty = Object.prototype.hasOwnProperty;
 
+const invariant = require('fbjs/lib/invariant');
```

Comment:
I guess you could use ES6 imports instead since the former ones do?

Reply:
Yes please!